iina/InfoDictionary.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  InfoDictionary.swift
+//  iina
+//
+//  Created by low-batt on 10/9/22.
+//  Copyright © 2022 lhc. All rights reserved.
+//
+
+import Foundation
+
+/// Wrapper for the dictionary constructed from the bundle’s `Info.plist` file.
+///
+/// This class exposes some of the entries contained in `Bundle.main.infoDictionary` as properties to provide for easier access
+/// to information contained in the dictionary from other classes.
+class InfoDictionary {
+  
+  static let shared = InfoDictionary()
+
+  let buildBranch: String?
+  let buildCommit: String?
+  let buildDate: String?
+  
+  let bundleIdentifier: String
+  
+  var copyright: String { Bundle.main.infoDictionary!["NSHumanReadableCopyright"] as! String }
+
+  var version: (String, String) {
+    let infoDic = Bundle.main.infoDictionary!
+    return (infoDic["CFBundleShortVersionString"] as! String, infoDic["CFBundleVersion"] as! String)
+  }
```

Comment:
Why are these two computed and the others aren't?

Reply:
Originally were stored properties. Then I began thinking reviewers might question why we are storing these values that are rarely referenced. So I converted these to match up with the `Utility` methods they replaced. I've pushed a commit that changes them all to be computed. If you would prefer they are all initialized once in an init method just let me know and I'll change it to be all stored properties.